src/ontology/mondo-edit.obo

```diff
@@ -230447,7 +230445,7 @@ xref: OMIM:300612 {source="DOID:0060829", source="Orphanet:3056", source="MONDO:
 xref: OMIM:300706 {source="MONDO:equivalentObsolete", source="DOID:0060811", source="Orphanet:85328", source="Orphanet:85328/e"}
 xref: OMIM:309590 {source="Orphanet:3056", source="MONDO:equivalentTo"}
 xref: Orphanet:3056 {source="DOID:0060829", source="MONDO:equivalentObsolete", source="OMIM:300612"}
-xref: Orphanet:85328 {source="OMIM:300706", source="MONDO:equivalentObsolete", source="OMIM:309590", source="DOID:0060811", source="MONDO:preferredExternal"}
+xref: Orphanet:85328 {source="OMIM:300706", source="MONDO:equivalentObsolete", source="OMIM:309590", source="DOID:0060811"}
```

Comment:
This term is not obsolete in orphanet: https://www.orpha.net/en/disease/detail/85328?name=85328&mode=orpha

Reply:
How does Mondo handle this statement:
```
This entity has been excluded from the Orphanet nomenclature of rare diseases and moved to [Non-specific syndromic intellectual disability](https://www.orpha.net/en/disease/detail/528084)?
```
And are you saying it should be `MONDO:equivalentTo`?

Reply:
I did not see the message that "This entity has been excluded from the Orphanet nomenclature of rare diseases and moved to [Non-specific syndromic intellectual disability](https://www.orpha.net/en/disease/detail/528084)?"
What you did, using "MONDO:equivalentObsolete" was correct. Thank you!

Reply:
Ok, just to note it was originally "MONDO:equivalentObsolete", that was not a change I made.

It's fixed now.